src/nomad_simulations/schema_packages/model_method.py

```diff
@@ -111,6 +111,445 @@ class ModelMethodElectronic(ModelMethod):
     )
 
 
+class FrozenCore(ArchiveSection):
```

Comment:
don't these have a parent class somehow? or it can be for later when more classes are defined?